build.gradle.kts

```diff
@@ -85,7 +85,7 @@ subprojects {
         withJavadocJar()
     }
 
-    if (!(project.name.contains("app") || project.path.contains("gradle"))) {
+    if (!(project.path.contains("gradle"))) {
```

Comment:
Was this an intentional change?

Reply:
Yes, because the application is now part of the app module, and the bootRun task is not included without the spring boot plugin.